python/py-prompt_toolkit/Portfile

```diff
@@ -26,6 +26,8 @@ if {${name} ne ${subport}} {
     depends_lib-append \
                     port:py${python.version}-wcwidth
 
+    patchfiles      patch-importlib-metadata.diff
```

Comment:
I am pretty sure this is not an issue with prompt_toolkit but something upstream aws-cli has to fix...

At least patch the file then in the "correct" way where you first put in a "@@VERSION@@" placeholder (or something similar) and the do a "reinplace" with the version number in the Portfile. That will avoid changing the patch-file when updating to a newer version.